frontend/src/app/AboutDialog.tsx

```diff
@@ -22,10 +28,40 @@ const AboutDialog: React.FC<AboutDialogProps> = ({ onClose }) => {
   const { isRHOAI } = useAppContext();
   const { theme } = useThemeContext();
   const { serverURL } = useClusterInfo();
+  const [dscStatus, loadedDsc, errorDsc] = useFetchDscStatus();
   const [dsciStatus, loadedDsci, errorDsci] = useFetchDsciStatus();
   const [subStatus, loadedSubStatus, errorSubStatus] = useWatchOperatorSubscriptionStatus();
   const error = errorDsci || errorSubStatus;
-  const loading = (!errorDsci && !loadedDsci) || (!errorSubStatus && !loadedSubStatus);
+  const loading =
+    (!errorDsci && !loadedDsci && !loadedDsc) || (!errorSubStatus && !loadedSubStatus && !errorDsc);
+
+  // Group components by display name while merging releases (excluding 'dashboard')
+  const groupedComponents = useMemo(() => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const componentMap: Record<string, { releases: any[] }> = {};
```

Comment:
Avoid `any` type and disabling eslint rules.
```suggestion
    const componentMap: Record<
      string,
      { releases: NonNullable<DataScienceClusterComponentKindStatus['releases']> } | undefined
    > = {};
```